server/plugins/irc-events/link.ts

```diff
@@ -78,8 +80,14 @@ function parseHtml(preview, res, client: Client) {
 	// TODO:
 	// eslint-disable-next-line @typescript-eslint/no-misused-promises
 	return new Promise((resolve: (preview: FetchRequest | null) => void) => {
-		const $ = cheerio.load(res.data);
-
+		let $;
```

Comment:
this probably doesn't need the conditional. Chances are you can pass undefined there as a transport encoding and be fine.

Reply:
> this probably doesn't need the conditional. Chances are you can pass undefined there as a transport encoding and be fine.

OK, makes sense, will try that